src/main/java/echoquery/sql/SingletonConnection.java

```diff
@@ -15,7 +15,7 @@
 
   private static final String URI = 
       "jdbc:mysql://speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com:3306/";
-  private static final String DB = "bestbuy";
+  private static final String DB = "airlines";
```

Comment:
hmm this breaks our tests which is querying over the much easier bestbuy dataset